littlelight.py

```diff
@@ -27,6 +27,12 @@ def main():
     menu = 'main'
 
     # buttons
+    newGame = button([width / 3, height / 2 + 50, width / 3, 40], gameDisplay, (0, 0, 0))

```

Comment:
I might be misreading the file, but it looks like the newGame button and cont button are defined twice, once on lines 30 and 31, and once on lines 36 and 37